README.md

```diff
@@ -117,6 +117,134 @@ class Test {
 ```
 
 
-## Future work
-  - Converters for `java.util.stream`
-  - [`Spliterator`](https://docs.oracle.com/javase/8/docs/api/java/util/Spliterator.html)s for Scala collections
+## Converters from Scala collections to Java 8 Streams
+
+Scala collections gain `seqStream` and `parStream` as extension methods that produce a Java 8 Stream
```

Comment:
I'm wondering if `seq` and `par` in Scala collections should be renamed to `sequential` and `parallel`. AFAICT they see little use in practice, `seq` is confusing (because we also have a `Seq` type) q